pkg/googlecloud/googlecloud.go

```diff
@@ -41,13 +40,18 @@ func NewClient(ctx context.Context, credentialsJSON []byte, projectID, instanceI
 		bigtableClient: bigtableClient,
 	}
 
+	return ClientBuilder(ctx, projectID, instanceID, &metricClientWrapped, &bigtableClientWrapped), nil
+}
+
+func ClientBuilder(ctx context.Context, projectID, instanceID string, metricClientWrapped MetricClientWrapper,
```

Comment:
WDYT about renaming this method? `ClientBuilder` seems to reference an object when in fact it is building the client.

```suggestion
func NewClientWithWrappers(ctx context.Context, projectID, instanceID string, metricClientWrapped MetricClientWrapper,
```

Reply:
I don't know, suggested to remove "wrapper" from names.
I personally don't see a problema in having wrappers, therefore I liked your suggestion.
What do you think @gabrielamafra ?

Reply:
I don't like the term "wrapper" either, I was just suggesting something to replace `ClientBuilder`.

Reply:
I think `NewClient` should be the standard constructor, which right now is this `ClientBuilder`. The other function, that receives only metadata and builds the clients, could be called `NewDefaultClient` or something more specific.

Reply:
I think `Default` can be a little misleading here, since both methods are no "default" but just require a different type of argument.

Reply:
My interpretation of `Default` here is "use the defaults for arguments I didn't pass", which would be the same as having optional arguments, but Go doesn't have such thing

Reply:
`Standard`?

Reply:
Yeah, the problem here is that all arguments are always required, that why I don't think about them as "default". They are just different.